### PR TITLE
Implement installation from custom packages.

### PR DESCRIPTION
**Pull request by jm404**
Hi team,

This PR implements the required vars to make use of `wazuh-ansible` installation from custom packages.

Best regards,

Jose

### CODE REVIEW

File: tests/system/provisioning/roles/aws/templates/ansible_hosts.j2 [hunk 1]
```diff
@@ -318,17 +343,8 @@ windowsagents:
...
-{% if packages_repository == 'staging' %}
-            repo: 'https://packages-dev.wazuh.com/pre-release/windows/'
-            md5: {{ windows_md5 }}
-{% elif packages_repository == 'devel' %}
-            repo: 'https://packages-dev.wazuh.com/staging/windows/'
-            md5: {{ windows_md5 }}
-{% endif %}
-{% endif %}
+        wazuh_winagent_config_url: https://s3-us-wes-1.amazonaws.com/packages-dev.wazuh.com/warehouse/branches/3.12/windows/wazuh-agent-3.12.0-0.3319fimreworksqlite.msi
+        wazuh_winagent_package_name: wazuh-agent-3.12.0-0.3319fimreworksqlite.msi
```
Comment (okynos) [inline tests/system/provisioning/roles/aws/templates/ansible_hosts.j2]: Why don't use `wazuh_winagent_config_url` and `wazuh_winagent_package_name` variables?

File: tests/system/provisioning/vars/wazuh_agent.yml [hunk 2]
```diff
@@ -10,3 +10,18 @@ windows_md5: "e4623e7cd3f2fc6ac8a313cbdd7c3cba"
 windows_repo: https://packages.wazuh.com/3.x/windows/
 windows_install_dir: C:\Program Files\ossec-agent\
 windows_install_dir_x86: C:\Program Files (x86)\ossec-agent\
+
+
+### Custom packages installation
+
+# Linux
+
+wazuh_custom_packages_installation_agent_enabled: false
```
Comment (okynos) [inline tests/system/provisioning/vars/wazuh_agent.yml]: Agent is false but manager is true, why?.

File: tests/system/provisioning/roles/aws/templates/ansible_hosts.j2 (hunk 1 above)
Comment (jm404) [reply tests/system/provisioning/roles/aws/templates/ansible_hosts.j2]: Done in 872f0b9b52219ef51a929b640f7f51eabf5fcfe4

File: tests/system/provisioning/vars/wazuh_agent.yml (hunk 2 above)
Comment (jm404) [reply tests/system/provisioning/vars/wazuh_agent.yml]: Done in 1d31a131b7e3dbabaafdbe6b314a78889da5619f

**Comment by jm404** [comment]
Merging, thanks @okynos !

**Pull request merged**